src/common/ScreenExporter.hpp

```diff
@@ -0,0 +1,58 @@
+/* *****************************************************************************
+ * A.L.E (Arcade Learning Environment)
+ * Copyright (c) 2009-2013 by Yavar Naddaf, Joel Veness, Marc G. Bellemare and 
+ *   the Reinforcement Learning and Artificial Intelligence Laboratory
+ * Released under the GNU General Public License; see License.txt for details. 
+ *
+ * Based on: Stella  --  "An Atari 2600 VCS Emulator"
+ * Copyright (c) 1995-2007 by Bradford W. Mott and the Stella team
+ *
+ * *****************************************************************************
+ *  ScreenExporter.hpp 
+ *
+ *  A class for exporting Atari 2600 frames as PNGs.
+ *
+ **************************************************************************** */
+
+#ifndef __SCREEN_EXPORTER_HPP__
+#define __SCREEN_EXPORTER_HPP__ 
+
+#include <string>
+#include "display_screen.h"
+#include "ale_screen.hpp"
+
+class ScreenExporter {
+
+    public:
+
+        /** Creates a new ScreenExporter which can be used to save screens using save(filename). */ 
+        ScreenExporter(ColourPalette &palette);
```

Comment:
I assume it takes ColourPalette as an argument to the constructor in case a user wants to use a non-standard palette. I think in most cases, people will just want to use the default palette. In this case it would be nice to have an empty constructor which would just construct a default color palette.